contracts/src/periphery/cert/interfaces/IEigenDACertVerifierBase.sol

```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+interface IEigenDACertVerifierBase {
+    /// @notice Check a DA cert's validity
+    /// @param abiEncodedCert The ABI encoded certificate. Any cert verifier should decode this ABI encoding based on the certificate version.
+    /// @return status An enum value. Success is always mapped to 1, and other values are errors specific to each CertVerifier.
+    function checkDACert(bytes calldata abiEncodedCert) external view returns (uint8 status);
+}
```

Comment:
Why do we need this as a separate Interface again? Its just because the router respects this?

Are all the other methods on the CertVerifier interface only used for the dispersal route? If so, then I suggest we instead rename the interfaces to `IEigenDACertVerifier` (this one) and `IEigenDACertBuilder` or something (the other one). See https://github.com/Layr-Labs/eigenda/pull/1532#discussion_r2084445291

Reply:
It's just set theory here. The router and the cert verifiers are all expected to expose this function, but no other functions are necessarily expected to be shared between these contracts.

I'm okay with any name, but I will point out that this sort of naming is pretty standard in the solidity world.